src/scenes/home/team/team.js

```diff
@@ -56,8 +56,10 @@ class Team extends Component {
             <div className={styles.foundingMembers}>
               <p>
                 Operation Code deeply appreciates the time, energy, and hard work of
-                our <b>Founding Board Members</b>: Aimee Knight, Laura Gomez, Pete
-                Runyon, Josh Carter, and Nick Frost.
+                our <b>Founding Board Members</b>, including Mark Kerr (Chair), Laura
+                Gomez (Vice Chair), Pete Runyon (Secretary/ Treasurer), Josh Carter,
+                Nick Frost, and Aimee Knight on their support, dedication and
+                commitment in the early days.
               </p>
```

Comment:
Resolved merge conflict, as `team.js` had some text updated in d9736d5 which hadn't been included here.